src/main/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtils.java

```diff
@@ -176,11 +188,30 @@ else if ( commonContigs.size() == dict1.size() && commonContigs.size() == dict2.
             return SequenceDictionaryCompatibility.IDENTICAL;
         else if ( ! commonContigsAreAtSameIndices(commonContigs, dict1, dict2) )
             return SequenceDictionaryCompatibility.DIFFERENT_INDICES;
+        else if ( supersets(commonContigs, dict1, dict2))
+            return SequenceDictionaryCompatibility.SUPERSETS;
         else {
             return SequenceDictionaryCompatibility.COMMON_SUBSET;
         }
     }
 
+
+    /**
+     * Utility function that tests whether dict2's set of contigs supersets dict1's
+     *
+     * @param commonContigs
+     * @param dict1
+     * @param dict2
+     * @return true if dict2's set of contigs supersets dict1's
+     */
+    private static boolean supersets(Set<String> commonContigs, SAMSequenceDictionary dict1, SAMSequenceDictionary dict2) {
+        Set<SAMSequenceRecord> setOne = (Set<SAMSequenceRecord>) dict1.getSequences();
+        Set<SAMSequenceRecord> setTwo = (Set<SAMSequenceRecord>) dict2.getSequences();
+        return setOne.containsAll(setTwo) && commonContigsHaveSameLengths(commonContigs, dict1, dict2);
```

Comment:
You don't need to call `commonContigsHaveSameLengths()` here, since the `containsAll()` call will invoke `equals()` on the `SAMSequenceRecord` objects, which will check both the names and the lengths.